packages/arb-token-bridge-ui/src/hooks/useAccountType.ts

```diff
@@ -10,12 +11,14 @@ type Result = {
 }
 
 export function useAccountType(): Result {
-  const { address } = useAccount()
   const { chain } = useNetwork()
+  const { address, isConnected } = useAccount()
 
   const { data: isSmartContractWallet = false, isLoading } = useSWRImmutable(
-    address && chain ? [address, chain.id, 'useAccountType'] : null,
-    ([_address, _chainId]) => addressIsSmartContract(_address, _chainId),
+    address && isConnected && chain
+      ? [address, chain.id, 'useAccountType']
+      : null,
+    async ([_address, chainId]) => addressIsSmartContract(_address, chainId),
```

Comment:
Why did we remove `_chainId`?

Reply:
there is no `chainId` so we don't have to use `_` to prevent collision so i renamed it to `chainId`